.circleci/config.yml

```diff
@@ -407,6 +407,14 @@ jobs:
       - run_test:
           pattern: 'opentracer'
 
+  opentelemetry:
```

Comment:
This is part of adding the lockfiles? If so, what were we doing before for Otel testing? I think I'm probably missing something?

Reply:
This is a merge screwup. I'll fix the diff. The only parts to review in this PR are the gitignore and `.riot/requirements`.